frontend/app/src/components/home/CurrentChatMessages.svelte

```diff
@@ -82,6 +82,8 @@
     $: chatStateStore = client.chatStateStore;
     $: userStore = client.userStore;
     $: isBot = chat.kind === "direct_chat" && $userStore[chat.them]?.kind === "bot";
+    // Show the avatar if the earliest loaded event is the earliest available event
```

Comment:
this seems to describe exactly the condition we already have i.e. that there are no more previous messages to load. Don't get why it needs to change.

Reply:
The issue is that the current check was based purely on confirmed events, so the avatar would hide, then reappear when the messages confirm.